versioned_docs/version-23.10/integrations/itsm/ot-easyvista-rest-api.md

```diff
@@ -0,0 +1,137 @@
+---
+id: ot-easyvista-rest-api
+title: EasyVista RestAPI
+---
+
+## How it works
```

Comment:
this short sentence does not actually explain how it works. it would be better to remove the title